rosidl_tutorials/rosidl_tutorials_msgs/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<package format="3">
```

Comment:
While the manifest file doesn't *need* format 3 it is probably good to bump it so users can start using format 3 -only features without having to worry about this. Please update the referenced schema file in the line above to match the format change.

Reply:
Done.